zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt

```diff
@@ -325,6 +325,7 @@ class ZiplineLoader internal constructor(
      * Fetch and receive ZiplineFile module.
      */
     suspend fun run() {
+      eventListener.moduleLoadStart(applicationName, id)
```

Comment:
I suspect this is going to be difficult to act upon well, cause users are going to measure this start/end pair but they really aught to measure download time + receive time.

What would you think about putting the module name into the downloadStart task, and just measuring modules as two independent steps?

(Also that leaks our implementation details less!)